docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+
+# Tracing
+
+If you use Serverless Console's AWS Lambda Extension for Node.js, it will automatically trace your AWS Lambda invocations, providing a timeline rich with information we curated to help you assess, optimize and troubleshoot, as well as trace services and data that triggered your AWS Lambda functions.
+
+Before the handler of your AWS Lambda function runs, our Extension loads our SDK and performs the auto-instrumentation logic.  This is what it looks like:
+
+```javascript
+import serverless from 'serverless-sdk-aws-lambda'
+
+serverless.instrument(params)
+
+// Your handler is called next...
+```
+
+Every AWS Lambda function invocation instrumented with our Extension generates a Trace.  This Trace contains the following Spans, some of which are optional depending on the modules you use within your code.  We put great effort into enriching this Trace with Tags useful for debugging across code, AWS Services and more.
+
+Here is a table of contents of Spans we currently capture, in an example hierarchical format:
+
+* **aws.lambda**
+  * **aws.lambda.initialization**
+  * **aws.lambda.invocation**
+    * **express**
+      * **node.https**
+      * **node.http**
+      * **aws.sdk**
+
+# Spans
+
+## `aws.lambda`
+
+This is the parent Span (aka the Trace) for an AWS Lambda. It measures the combined lifecyle phases of AWS Lambda Initialization and Invocation, and any logic performed within the Invocation phase.
+
+Additionally, the duration of this Span is what AWS Lambda bills for, based on 1ms increments. Duration charges apply to initialization code that is declared outside of the handler in the Initiatlization phase, code that runs in the handler of a function during the Invocation phase, as well as the time it takes for code in any last running Extensions to finish executing during Shutdown phase.
+
+It’s important to note that duration of this Span for AWS Lambda is not the same as the performance your users and customers experience when using your AWS Lambda-based application. The Spans of AWS Lambda Initialization and Invocation duration affect your application experience, not the AWS Lambda Shutdown.
+
+### Tags
+
+These are the Tags attached to this Span:
+
+```javascript
+/* Tags: Standard */
+
+s.sdk.name: "s-aws-lambda-nodejs"
+s.sdk.version: "0.0.1"
+s.org.id: "53121673-361f-4181-3411-53e031312c09"
+s.platform: "aws-lambda"
+s.environment: "prod"
+s.namespace: "api"
+s.service: "aws-api-prod-getPoster"
+s.region: "us-east-1"
+s.request.id: "2be6c182-955a-4da9-9c39-d9e9d9febbaa" // Identifies this unique request
+s.trace.id: "db8e08ae5e6c71c35320ddad5ffb56a5"
+s.span.id: "e8a4e2a264145041" // This Span's ID
+s.span.timestamp: "2022-07-13T20:10:48.794945024Z" // This Span's time.  Same as start_time.
+s.span.start_time: "2022-07-13T20:10:48.794945024Z" // This Span's start time.
```

Comment:
Are we sure we want to follow `snake_case` naming convention instead of `camelCase` which is common in JS and JSON?

Reply:
Yes, this tag convention will be used across all languages.